.packit.yml

```diff
@@ -0,0 +1,37 @@
+# Docs: https://packit.dev/docs/
```

Comment:
Strangely enough, https://packit.dev/docs/ says `Page Not Found`, while https://packit.dev/docs works.

Reply:
Fixed